src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,13 @@
+package duke;
```

Comment:
Is there a reason this class is being tracked on revision control? This feels like a class created to test your gradle test functionalities rather than code meant for your product so I think this isn't meant to be included?